### PR TITLE
monitoring-logs-doc-updated

### PR DESCRIPTION
**Pull request by Buddhimah**
This PR depends of the fix for the issue : https://github.com/wso2/product-is/issues/6696

### CODE REVIEW

File: en/docs/administer/monitoring-logs.md [hunk 1]
```diff
@@ -29,11 +29,7 @@ the `          <IS_HOME>/repository/logs         ` directory.
         mentioned scenario to plug the handlers and appenders. The following
         default log4j2 appenders in the
         `            log4j2.properties           ` file are used for this
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: If we are removing the content mentioning the appenders, we should remove this sentence from the content as well right? 

"The following default log4j2 appenders in the  `            log4j2.properties           ` file are used for this"

File: en/docs/administer/monitoring-logs.md [hunk 2]
```diff
@@ -59,39 +55,45 @@ See the following information on configuring **Carbon logs**, **Audit
...
+appender.CARBON_LOGFILE.strategy.max = 20
+appender.CARBON_LOGFILE.filter.threshold.type = ThresholdFilter
+appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
+```
+
+The log levels that
+can be configured are [listed below](#MonitoringLogs-log4j-levels).
+
+!!! note
+    All the changes that are made to the log4j2.properties will be apply at run time.
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: please change "will be apply" to "will apply" or "will be **applied**"

File: en/docs/administer/monitoring-logs.md [hunk 3]
```diff
@@ -104,33 +106,38 @@ appender.AUDIT_LOGFILE.policies.type = Policies
...
 below, the root log level (INFO) will apply to all appenders by default.
 
+Following is how the log level is set to `DEBUG` for the `CARBON_LOGFILE` appender ([Carbon log](#configuring-carbon-logs)): 
 
 !!! example  
-
-    Following is how the log level is set to `DEBUG` for the `CARBON_LOGFILE` appender ([Carbon log](#configuring-carbon-logs)): 
-    ``` java
+   
     appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: Shall we put this inside a code block? 

appender.CARBON_LOGFILE.filter.threshold.level = DEBUG

To put it inside a code block, add ``` to the beginning and end

File: en/docs/administer/monitoring-logs.md [hunk 4]
```diff
@@ -152,70 +159,81 @@ Following are the log levels that can be configured:
...
-    -   `            log4j.appender.AUDIT_LOGFILE.MaxFileSize=10MB           `
-    -   `            log4j.appender.AUDIT_LOGFILE.MaxBackupIndex=20           `
+1.  Remove the
+       ``` toml
+       appender.AUDIT_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+       appender.AUDIT_LOGFILE.policies.time.interval = 1
+       ```
+    from the `AUDIT_LOGFILE` appender in the
+    `          <IS_HOME>/repository/conf/log4j2.properties         `
+    add the the following to the `AUDIT_LOGFILE` appender :
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: "the the" has been repeated. remove one "the"

File: en/docs/administer/monitoring-logs.md [hunk 5]
```diff
@@ -152,70 +159,81 @@ Following are the log levels that can be configured:
...
+       ``` toml
+    appender.AUDIT_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+    appender.AUDIT_LOGFILE.policies.size.size=10MB
+       ```
+2.  Following property under `AUDIT_LOGFILE` appender is to limit the number of backup files.
+    You can change it as per your requirement by changing its value.
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: please change to "**The** following property  under the `AUDIT_LOGFILE` appender is **used** to...."

File: en/docs/administer/monitoring-logs.md (hunk 1 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

File: en/docs/administer/monitoring-logs.md (hunk 2 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

File: en/docs/administer/monitoring-logs.md (hunk 3 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

File: en/docs/administer/monitoring-logs.md (hunk 5 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

File: en/docs/administer/monitoring-logs.md (hunk 4 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

File: en/docs/administer/monitoring-logs.md [hunk 6]
```diff
@@ -59,39 +53,45 @@ See the following information on configuring **Carbon logs**, **Audit
 logs,** **HTTP access logs**, and **Service/Event logs** for WSO2 Identity Server.
 
 #### Configuring Carbon logs 
-You can easily configure Carbon logs using the WSO2 Identity Server Management Console or you can manually edit the
+Carbon logs are enabled in WSO2 Identity Server by default. You can change
+the following default configuration by manually updating the the
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: the word "the" has been repeated twice in this sentence. let's remove one "the" please

File: en/docs/administer/monitoring-logs.md [hunk 7]
```diff
@@ -152,70 +159,81 @@ Following are the log levels that can be configured:
...
-    `            MaxFileSize           ` property, the content is copied
+    `            appender.CARBON_LOGFILE.policies.size.size           ` property, the content is copied
     to a backup file and the logs are continued to be added to a new
-    empty log file. The `            MaxBackupIndex           ` property
-    makes the Log4j maintain a maximum number of backup files for the
-    logs.
+    empty log file.
+    
+
+2.  The Following property under `CARBON_LOGFILE` appender is used to limit the number of backup files.
```
Comment (sherenewso2) [inline en/docs/administer/monitoring-logs.md]: "Following" should start with simple "f"

File: en/docs/administer/monitoring-logs.md (hunk 5 above)
Comment (sherenewso2) [reply en/docs/administer/monitoring-logs.md]: "Following" should start with simple "f"

File: en/docs/administer/monitoring-logs.md (hunk 6 above)
Comment (Buddhimah) [reply en/docs/administer/monitoring-logs.md]: done

**Pull request merged**